.circleci/osx_install_dependencies.sh

```diff
@@ -48,6 +48,8 @@ function validate_checksum {
 
 if [ ! -f /usr/local/lib/libz3.a ] # if this file does not exists (cache was not restored), rebuild dependencies
 then
+  brew uninstall temurin17
```

Comment:
This seems pretty random. Why does it fix things?

I'm not up to date with Matrix discussions for today yet - I guess I'll probably find some context there, but we should really include more context in PRs. "Fix issue" is not enough.

Reply:
Thanks!